### ISSUE TITLE
Bug laravel tag inside tag html

### ISSUE THREAD

**Issue by ferrianes** [EmranMR/tree-sitter-blade#69]
sorry, i think i have bugs when i insert @class inside html tag

![Screenshot 2024-08-19 170534](https://github.com/user-attachments/assets/34290a9a-b976-4d2e-9438-ffbd56911225)

**Comment by EmranMR** [comment]
Hi @ferrianes sorry for the delay, I ran a quick test on a `@class` for the following code and the parser seem to be working fine. can you try and isolate the error, in a smallest code snippet possible and I will run some tests.

```html
<a href="" @class('asd')>
	testing
</a>
```
![Screenshot 2024-08-31 at 21 41 15](https://github.com/user-attachments/assets/14e53f2b-18a1-4660-8413-641f407cc7a3)

**Comment by calebdw** [comment]
The error is likely coming from another parser like `html`

**Comment by ferrianes** [comment]
> Hi @ferrianes sorry for the delay, I ran a quick test on a `@class` for the following code and the parser seem to be working fine. can you try and isolate the error, in a smallest code snippet possible and I will run some tests.
> 
> ```
> <a href="" @class('asd')>
> 	testing
> </a>
> ```
> 
> ![Screenshot 2024-08-31 at 21 41 15](https://private-user-images.githubusercontent.com/11975985/363440180-14e53f2b-18a1-4660-8413-641f407cc7a3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzNDYzMTAsIm5iZiI6MTcyNTM0NjAxMCwicGF0aCI6Ii8xMTk3NTk4NS8zNjM0NDAxODAtMTRlNTNmMmItMThhMS00NjYwLTg0MTMtNjQxZjQwN2NjN2EzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDA2NDY1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjMTNmZjkyNTE5ODc5MTdlMWFiYTJlYTRmMjkzZjdlMjlkZjQ5YzRiNjNiNTQ5ZDgwYjRlZjIzMTZjNjY0ZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nzjYtTFHCrgMwi0fbExCvf6gjNqaF6sgtvRGDglyIAU)

yapp, still error for me, i dont know why
![image](https://github.com/user-attachments/assets/aca7bf39-a23d-4606-8083-1f86c9f2adca)

here is my config

html/injections.scm
```
;; extends

; AlpineJS attributes
(attribute
  (attribute_name) @_attr
    (#lua-match? @_attr "^x%-%l")
  (quoted_attribute_value
    (attribute_value) @injection.content)
  (#set! injection.language "javascript"))
(attribute
  (attribute_name) @_attr
    (#lua-match? @_attr "^:%l")
  (quoted_attribute_value
    (attribute_value) @injection.content)
  (#set! injection.language "javascript"))

; Blade escaped JS attributes
; <x-foo ::bar="baz" />
(element
  (_
    (tag_name) @_tag
      (#lua-match? @_tag "^x%-%l")
  (attribute
    (attribute_name) @_attr
      (#lua-match? @_attr "^::%l")
    (quoted_attribute_value
      (attribute_value) @injection.content)
    (#set! injection.language "javascript"))))

; Blade PHP attributes
; <x-foo :bar="$baz" />
(element
  (_
    (tag_name) @_tag
      (#lua-match? @_tag "^x%-%l")
    (attribute
      (attribute_name) @_attr
        (#lua-match? @_attr "^:%l")
      (quoted_attribute_value
        (attribute_value) @injection.content)
      (#set! injection.language "php_only"))))
```

blade/injections.scm
```
((text) @injection.content
    (#not-has-ancestor? @injection.content "envoy")
    (#set! injection.combined)
    (#set! injection.language php))

; tree-sitter-comment injection
; if available
((comment) @injection.content
 (#set! injection.language "comment"))

; could be bash or zsh
; or whatever tree-sitter grammar you have.
((text) @injection.content
    (#has-ancestor? @injection.content "envoy")
    (#set! injection.combined)
    (#set! injection.language bash))

((php_only) @injection.content
    (#set! injection.language php_only))

((parameter) @injection.content                                                                                                 
    (#set! injection.include-children) ; You may need this, depending on your editor e.g Helix                                                                                          
    (#set! injection.language php_only))
```

**Comment by EmranMR** [comment]
Hi @ferrianes, As Caleb pointed out, it is probably the HTML thing. As long as the injection and highlighting is working fine, I would not worry about it. 
I just gave this a shot in Helix as well, and it is working fine. :)

**Comment by ferrianes** [comment]
okay i understand, but how i can fix that
as you mention in https://github.com/tree-sitter/tree-sitter-html/issues/65#issue-1861742008, how you fix that

**Comment by EmranMR** [comment]
That was resolved by [#27](https://github.com/EmranMR/tree-sitter-blade/issues/27)
and it seems to be working on my side, with Nova and Helix.

<img width="544" alt="image" src="https://github.com/user-attachments/assets/b9ffe0e4-330f-41ec-b217-f468ea79edcd">

**Comment by ferrianes** [comment]
Okey thanks bro, later i try again, thanks for feedback
